tests/functional/spec/census-household.spec.js

```diff
@@ -83,7 +83,7 @@ describe('Census Household', function () {
 
         // who-lives-here
         PermanentOrFamilyHome.clickPermanentOrFamilyHomeAnswerYes().submit()
-        HouseholdComposition.setPersonName(0, 'John Smith').submit()
+        HouseholdComposition.setFirstName(0, 'John').submit()
```

Comment:
The Jina filter I created should display middle and last names appropriately if only middle and/or last were provided. But the fact that we've added mandatory validation here will help to ensure that we always have something, which is good.